contracts/eos/Common/common.hpp

```diff
@@ -25,51 +25,64 @@ struct converter {
 
 struct memo_structure {
     path              path;
-    vector<converter> converters;   
+    vector<converter> converters;
     string            version;
     string            min_return;
     string            dest_account;
+    string            sender_account;
+    string            affiliate_account;
+    string	          affiliate_fee;
```

Comment:
minor indentation issue

Reply:
It doesn't appear as such in my editor

Reply:
I just changed it to have 1 space after the type everywhere